src/store.js

```diff
@@ -236,6 +246,8 @@ function EvolvStore(options) {
       }
     });
 
+    context.set('keys.active', arrays.from(configKeyStates.active))
```

Comment:
Rather than polyfilling arrays.from, why not just do:
const active = [];
configKeyStates.active.forEach(function(v) {
active.push(v);
});

or 

const active = [];
configKeyStates.active.forEach(active.push.bind(active));

Reply:
I think its nice to have the polyfill in case we need to do something similar in the future.